### PR TITLE
add owner check for bug

### PR DESCRIPTION
**Pull request by thotz**

### CODE REVIEW

**Comment by nixpanic** [comment]
👍 looks good to me. Note that not everyone can change the ownership of bugs, but sufficient people in #gluster-dev should be able to do so. For other contributors we can request additional privileges.

**Comment by nigelbabu** [comment]
Has this been tested?

File: compare-bug-version-and-git-branch.sh [hunk 1]
```diff
@@ -42,18 +42,24 @@ while [ -z "${BUG_PRODUCT}" -a ${BZQTRY} -le 3 ]; do
...
+    BUG_OWNER=$(cut -d: -f5 <<< "${BZQOUT}")
 done
 
 if [ "${BUG_STATUS}" != "NEW" -a "${BUG_STATUS}" != "POST" -a "${BUG_STATUS}" != "ASSIGNED" ]; then
     echo "BUG id ${BUG} has an invalid status as ${BUG_STATUS}. Acceptable status values are NEW, ASSIGNED or POST."
     exit 1
 fi
 
+if [ "${BUG_OWNER}" = "bugs@gluster.org" ]; then
+    echo "BUG id ${BUG} has an invalid owner"
```
Comment (nigelbabu) [inline compare-bug-version-and-git-branch.sh]: Could you change this message to: "BUG id ${BUG} is owned by bugs@gluster.org. Please assign it to yourself. If you do not have the right permissions, please ask the component owner for help".

**Comment by nigelbabu** [comment]
@thotz Could you address the review comments, please?

**Comment by thotz** [comment]
@nigelbabu I have tested the change long back. I have updated the message as you suggested

**Comment by nigelbabu** [comment]
What happened to the bit `Please assign it to yourself. If you do not have the right permissions, please ask the component owner for help`

**Comment by thotz** [comment]
I don't have the permission to do so. @nigelbabu can u please assign the PR to me?

**Comment by nigelbabu** [comment]
Closing in favor of a #184

**Pull request closed without merging**